internal/dag/dag.go

```diff
@@ -191,6 +191,9 @@ type UpstreamValidation struct {
 	SubjectName string
 }
 
+// CACertificateKey stores the key name for the TLS validation secret cert.
+const CACertificateKey = "ca.crt"
```

Comment:
Let's put this in `internal/dag/secret.go`, where it's closer to the subject matter.

Reply:
Now moved.